<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demography Projection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>891</width>
         <height>454</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="NewProject"/>
    <addaction name="OpenProject"/>
    <addaction name="separator"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionCloseProject"/>
    <addaction name="SearchObject"/>
    <addaction name="AddObject"/>
    <addaction name="separator"/>
    <addaction name="CloseProject"/>
    <addaction name="Quit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="DataManage">
    <property name="title">
     <string>基础数据</string>
    </property>
    <addaction name="InputData"/>
    <addaction name="ImportData"/>
    <addaction name="separator"/>
    <addaction name="WatchData"/>
    <addaction name="ModifyData"/>
   </widget>
   <widget class="QMenu" name="Parameter">
    <property name="title">
     <string>基本参数</string>
    </property>
    <addaction name="InputPara"/>
    <addaction name="ImportPara"/>
    <addaction name="separator"/>
    <addaction name="CalculatePara"/>
    <addaction name="separator"/>
    <addaction name="WatchPara"/>
    <addaction name="ModifyPara"/>
   </widget>
   <widget class="QMenu" name="Simulation">
    <property name="title">
     <string>政策仿真</string>
    </property>
    <addaction name="CalculateSim"/>
    <addaction name="ShowSim"/>
    <addaction name="separator"/>
    <addaction name="EvaluateSim"/>
    <addaction name="ReportSim"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="About"/>
   </widget>
   <addaction name="File"/>
   <addaction name="DataManage"/>
   <addaction name="Parameter"/>
   <addaction name="Simulation"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="NewProject">
   <property name="text">
    <string>新建预测或对象</string>
   </property>
  </action>
  <action name="OpenProject">
   <property name="text">
    <string>打开预测或对象</string>
   </property>
  </action>
  <action name="SearchObject">
   <property name="text">
    <string>查找研究对象</string>
   </property>
  </action>
  <action name="AddObject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>添加研究对象</string>
   </property>
  </action>
  <action name="CloseProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关闭项目</string>
   </property>
  </action>
  <action name="Quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="InputData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>录入</string>
   </property>
  </action>
  <action name="ImportData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="WatchData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </action>
  <action name="ModifyData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </action>
  <action name="InputPara">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>录入</string>
   </property>
  </action>
  <action name="ImportPara">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="CalculatePara">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </action>
  <action name="WatchPara">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </action>
  <action name="ModifyPara">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </action>
  <action name="CalculateSim">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>仿真计算</string>
   </property>
  </action>
  <action name="ShowSim">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>仿真演示</string>
   </property>
  </action>
  <action name="EvaluateSim">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>仿真评价</string>
   </property>
  </action>
  <action name="ReportSim">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>仿真报告</string>
   </property>
  </action>
  <action name="About">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>closeProject</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>saveProject</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
